README.md

```diff
@@ -43,9 +43,15 @@ class PersonTest {
 
     @Test
     fun testAge() {
-        assertThat("age", person.age).isGreaterThan(20)
```

Comment:
The order of arguments seemed to be wrong.

Reply:
Good catch! Forgot to update the README when we changed the order.